lib/frame.rb

```diff
@@ -0,0 +1,35 @@
+class Frame
+  attr_reader :frame_score, :current_roll, :rolls_remaining
+  attr_accessor :closed
+
+  def initialize 
```

Comment:
Everything looks good the methods are short and concise(names are telling the story, what they do)